applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -366,9 +308,9 @@ static void syncable_cb(struct bt_bap_broadcast_sink *sink, bool encrypted)
 
 	LOG_DBG("Broadcast sink is syncable");
 
-#if (CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED)
-	strncpy(bis_encryption_key, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY, 16);
-#endif /* (CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED) */
+	if (IS_ENABLED(CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED)) {
+		memcpy(bis_encryption_key, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY, 16);
```

Comment:
It was changed because of a compiler warning. As long as the BT_AUDIO_BROADCAST_ENCRYPTION_KEY is 16 characters long it should be the same. Might have to add some additional code here to handle shorter cases